### PR TITLE
Add new CLI commands, more internationalization, small fixes

### PR DESCRIPTION
**Pull request by AleksandrSokolov**
1. New CLI commands.
2. Improved Internationalization.
3. Updated Russian translation.
4. Small bug fixes

### CODE REVIEW

File: packages/saltcorn-data/base-plugin/viewtemplates/viewable_fields.js [hunk 1]
```diff
@@ -533,7 +533,7 @@ const get_viewable_fields = (
           );
           const label = column.action_label_formula
             ? eval_expression(column.action_label, r)
-            : __(column.action_label) || column.action_name;
+            : __(column.action_label) || __(column.action_name);
```
Comment (pyhedgehog) [inline packages/saltcorn-data/base-plugin/viewtemplates/viewable_fields.js:536]: What if translation not exists?

**Review by pyhedgehog — CHANGES_REQUESTED** [review]

**Comment by AleksandrSokolov** [comment]
@pyhedgehog I made update accroding your comments with some adtional improvements. Please see.

**Comment by pyhedgehog** [comment]
> @pyhedgehog I made update accroding your comments with some adtional improvements. Please see.

1. You've forgot to revert help change in `packages/saltcorn-cli/src/commands/install-pack.js`.
2. I've checked on sqlite3 (i.e. non-multitenant) installation and `run-trigger` works.
3. I've checked `run-sql` and it works.

**Comment by angemarius939** [comment]
Generate JavaScript function code for this case: 


If I select one district among: Gakenke, Rulindo, Nyarugenge district in another variable bring only corresponding sectors to district selected: sectors for Gakenke: Rushashi, Muyongwe,Minazi. The sectors for Rulindo district are: Tare, Shyorongi, Kinihira. The sectors for Nyarugenge District are: Rugarama, Muhima, Kigali.

**Comment by angemarius939** [comment]
Generate JavaScript function code for this case: 


If I select one district among: Gakenke, Rulindo, Nyarugenge district in another variable bring only corresponding sectors to district selected: sectors for Gakenke: Rushashi, Muyongwe,Minazi. The sectors for Rulindo district are: Tare, Shyorongi, Kinihira. The sectors for Nyarugenge District are: Rugarama, Muhima, Kigali.

**Comment by AleksandrSokolov** [comment]
@glutamate Please review and help with CI

**Comment by glutamate** [comment]
@AleksandrSokolov Thanks for this. I have been away - will look later today

**Comment by glutamate** [comment]
Thanks, this is very useful, especially run-sql will save a lot of time. Somethings may be improved by switching to console.table for outputting lists of things, but I think this is a general comment for a lot of the CLI cmds so we can do that later

**Pull request merged**